Unit Tests/Tests/Authentication/Token Providers/TestTokenProviderTests.swift

```diff
@@ -0,0 +1,173 @@
+import XCTest
+import Mockingjay
+@testable import PusherChatkit
+
+class TestTokenProviderTests: XCTestCase {
+    
+    // MARK: - Properties
+    
+    var testURL: URL!
+    var matcher: (URLRequest) -> Bool = uri("https://instance.pusherplatform.io/services/chatkit_token_provider/test/locator/token?user_id=testUserID")
```

Comment:
I don't think you need to give this an initial value because you're always setting it in the `setUp` method.

Reply:
I forgot to remove this initialization.

Reply:
👀  :-)